release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -743,6 +743,24 @@ Configuring an Ingress Controller to inject an HTTP header with a uniquely defin
 
 For more information, see the xref:../rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc#specification[`IngressController` specification].
 
+[id="ocp-4-6-network-policy-host-network-ingress-controllers"]
+==== Network policy supports selecting host network Ingress Controllers
+
+When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
+In a network policy rule, the `network.openshift.io/policy-group: ingress` namespace selector matches traffic from an Ingress Controller.
```

Comment:
I think we want to somehow say that while this legacy label (`network.openshift.io/policy-group: ingress`) would work for OVN-K as well, we really want folks to start using `"policy-group.network.openshift.io/ingress=""` as the ingress namespace selector.

Reply:
Actually, for 4.6, for OpenShift SDN, we didn't backport the treating of openshift-host-network namespace as the one to use for ingress traffic. Neither did we update the openshift-ingress namespace to carry the new label. So may be the 4.6 documentation is OK as is. But for 4.7 and 4.8 we want to have folks use the new labels consistently across both SDN and OVN-K.

Reply:
@abhat, does that mean I need to omit _OpenShift SDN_ from these release notes for this? As time has passed since the release, my recall on this feature seems to be drifting some.

Reply:
For 4.6, yes. for 4.7 and 4.8, no you don't need to omit. For 4.6 SDN, we should let folks continue using the old label for ingress.